subprojects/docs/src/docs/release/notes.md

```diff
@@ -117,24 +103,10 @@ testing {
 }
 ```
 
-#### Dependency verification metadata improvements
```

Comment:
@asodja can you add more context to this section, why is it helpful for a developer?

Reply:
Sure, will add.

Reply:
I added reviews suggestions down below.